thanos/charts/values.yaml

```diff
@@ -152,6 +152,26 @@ thanos:
         # tlsCert:
         # tlsKey:
 
+    ## Expose Thanos Query as a Plutono datasource
+    ##
+    plutonoDatasource:
+      # -- Creates a Perses datasource for standalone Thanos Query
+      create: false
+
+      # -- Label selectors for the Plutono sidecar to detect this datasource.
+      selector: {}
+        # plutono-global: '"true"'
```

Comment:
Could you tell, How this selector will work? 
from my understanding, Plutono will detect all the configmaps with the labels mentioned in [plutono/charts/values.yaml](https://github.com/cloudoperators/greenhouse-extensions/blob/thanos%3Bcreate-plutono-perses-datasource/plutono/charts/values.yaml#L877-L879

Reply:
Same for Plutono.